lib/rgit/cli.rb

```diff
@@ -282,7 +282,35 @@ def init(directory = Pathname.pwd)
     desc "test", "A test function"
     def test(*args)
       j = args.class
+      puts j
+      puts args
     end
+
+    desc "log", "Show commit logs"
+    def log(*args)
+      pass_through_cmd("log", args)
+    end
+
+    desc "status", "Show the working tree status"
+    def status(*args)
+      git_output = pass_through_cmd("status", args)
+      out = git_output.partition(%r{@[^@]*/%.*}) # isolate branchname
+      out[1] = out[1][1..-1].partition("/")[0] if out[1] != ""
+      # replace w/ group_name - can't use put since thor does overriding of it
+      puts out.join()
```

Comment:
Do not use parentheses for method calls with no arguments.<br>Do not write to stdout. Use Rails' logger if you want to log.